resources/js/src/app/store/modules/SingleItemModule.js

```diff
@@ -111,9 +111,16 @@ const getters =
         {
             const graduatedPrice = getters.variationGraduatedPrice ? getters.variationGraduatedPrice.unitPrice.value : 0;
 
+            if (graduatedPrice === null)
+            {
+                return null;
+            }
+
             if (state.variation.documents)
```

Comment:
Change to `if (!isNullOrUndefined(graduatedPrice) && state.variation.documents) { ... }`

Reply:
Requires `import {isNullOrUndefined}from "../../helper/utils";` at first line